source/composeK.js

```diff
@@ -39,5 +38,9 @@ export default function composeK() {
   }
   var init = Array.prototype.slice.call(arguments);
   var last = init.pop();
-  return compose(compose.apply(this, map(chain, init)), last);
+  if (init.length === 0) {
+    last;
```

Comment:
Should this have a return in front of it?

Reply:
Definitely! Should I make a change to re-make a pull request?